StyleCop.Analyzers/StyleCop.Analyzers.Test/DocumentationRules/SA1613UnitTests.cs

```diff
@@ -0,0 +1,124 @@
+﻿namespace StyleCop.Analyzers.Test.DocumentationRules
+{
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Microsoft.CodeAnalysis.Diagnostics;
+    using Analyzers.DocumentationRules;
+    using TestHelper;
+    using Xunit;
+
+    /// <summary>
+    /// This class contains unit tests for <see cref="SA1617VoidReturnValueMustNotBeDocumented"/>-
```

Comment:
:exclamation: Incorrect reference, and for some reason it ends with a `-` instead of `.`.